src/tx/applyTx/checkEpochLength.js

```diff
@@ -21,4 +21,5 @@ module.exports = (state, tx, bridgeState) => {
 
   state.epoch.epochLengthIndex += 1;
   state.epoch.epochLength = tx.options.epochLength;
+  bridgeState.epochLength = state.epoch.epochLength;
```

Comment:
Why not like this?
```
state.epoch.epochLength = tx.options.epochLength;
bridgeState.epochLength = tx.options.epochLength;
```
_(It's more about whole method, rather than specific change)
p.s. I thought that using impure functions is a bad practice..._